counting_sort/counting_sort.py

```diff
@@ -0,0 +1,38 @@
+#Running time Complexity: O(n)
+#Auxillary Space: O(n)
+def count_occurences(a, c):
+    """
+    Input params: Array, intermediate variable
```

Comment:
@khilanravani Thats not the correct way for the docstrings please see other python codes for example